superset/config.py

```diff
@@ -256,8 +256,9 @@
 
 # CSV Options: key/value pairs that will be passed as argument to DataFrame.to_csv method
 # note: index option should not be overridden
+
 CSV_EXPORT = {
-    'encoding': 'utf-8',
+    'encoding': 'utf-8-sig',
```

Comment:
what are the implications of this?

Reply:
If utf8 is used, The result will be unintelligible text on a Windows platform. If gbk is used,The result will be unintelligible text on the Linux platform or Mac paltform ,If utf-8-sig is used .They can display normal